docs/flow-visibility.md

```diff
@@ -0,0 +1,45 @@
+#Flow Visibility in Antrea
+
+[Antrea](architecture.md) implements Container Network Interface(CNI) and Network Policy (both Kubernetes and customized)
+ to provide network connectivity and security features for pod workloads. Considering the enormity and dynamism of Kubernetes
+ workloads in a cluster, Flow Visibility helps in the management and configuration of Kubernetes constructs such as Network
+ Policy, Services, Pods etc., and there by enhance the performance and security aspects of pod workloads.
```

Comment:
s/and there by enhance the performance and security aspects of pod workloads./and thereby provides opportunities to enhance the performance and security aspects of Pod workloads.